claims_hosp/HospClaims/automate/ftp_to_covidcast.py

```diff
@@ -0,0 +1,62 @@
+"""FTP created files over to Delphi Covidcast ingestion."""
+# standard
+import datetime
+import os
+from pathlib import Path
+
+# third party
+import click
+import paramiko
+
+# first party
+from secrets import covidcast
+
+NUM_FILES = 71*6*2  # expect (71 dates x 6 geos x 2 signals)
```

Comment:
If https://github.com/cmu-delphi/covidcast-indicators/pull/1419 lands first, then this should be changed to `70*6*2`, as we don't shift dates forward any longer, which results in 1 less day produced.